docs/architecture/zero-token-nodes.rst

```diff
@@ -0,0 +1,15 @@
+=========================
+Zero-token Nodes
+=========================
+
+By default, all nodes in a cluster own a token range and are used to replicate
+data. In certain circumstances, you may choose to add a node that doesn't own
+any token. Such nodes are referred to as zero-token nodes. They do not have
+a copy of the data but only participate in Raft quorum voting.
```

Comment:
They can also serve as coordinators. A present, CQL drivers will **not** send requests to zero-token nodes, but Alternator clients can be configured to send requests to such a node; In this capacity a zero-token node is sometimes called a "coordinator-only node" or "proxy nodes" (https://image.slidesharecdn.com/cassandrasummit2016-160913041336/95/cassandra-at-instagram-2016-dikang-gu-facebook-cassandra-summit-2016-17-638.jpg?cb=1473961469)

Reply:
@nyh While true, I do not think we should include this in the docs for now.
Once and if we create such a solution, we can revisit it.